web/massastation/src/mirage/server.js

```diff
@@ -88,6 +139,10 @@ function mockServer(environment = ENV.DEV) {
 
         return plugin.destroy();
       });
+      this.get('plugin-store', (schema) => {
+        let { models: stores } = schema.stores.all();
+        return stores;
+      });
```

Comment:
```suggestion

      this.get('plugin-store', (schema) => {
        let { models: stores } = schema.stores.all();
        
        return stores;
      });
```

Reply:
we should configure prettier to fix these blank lines

Reply:
Totally agree! No doubts! BTW the console.log we add is not working neither!